lpcompiler.pas

```diff
@@ -1653,13 +1662,17 @@ function TLapeCompiler.ParseMethod(FuncForwards: TLapeFuncForwards; FuncHeader:
               addMethod(OldDeclaration as TLapeGlobalVar);
             end;
 
+            MethodDef := FuncHeader.MethodDef;
```

Comment:
The idea here was that when / if we have to create the OverloadMethod type to host functions we'd also store what kind of methods it holds, if it's property, functions or operators.. So 8 lines further down we can check if the overloadMethod already holds some other type of method.

This is to avoid / disallow:
`> function TFoo.Foo(...): Type; overload;`
`> property TFoo.Foo(...): Type;`